test/test-cases/regression/rule-920450.json

```diff
@@ -0,0 +1,87 @@
+[
```

Comment:
The file name for issues follow the pattern - issue-number.json.

We do have some **rule-**, but we better keep the issue related to it, so the user will able to link the test with the correspondent background information on it.

The issue number is #2296 opened by @michaelgranzow-avi

Reply:
I changed it in my last commit ([ceb0e3](https://github.com/SpiderLabs/ModSecurity/pull/2297/commits/ceb0e381d0338442b6d4761c9847630202611673)).

Just a small note: this issue (#2296) was reported by @michaelgranzow-avi, but I sent the first PR in June of 2019 (#2107) where I described this issue.